app/views/research_sessions/_share.html.erb

```diff
@@ -0,0 +1,27 @@
+<div class="share">
+  <div class="share__button">
+    <button class="button button--small button--green button--no-margin js-share-trigger">Share</button>
+  </div>
+  <div class="share__content js-share-target">
+    <p>Here is the link you can share and use at any time:</p>
+
+    <div class="share__url">
+      <input
+        type="text"
+        id="share-input"
+        class="share__input"
+        value="<%= request.original_url %>"
+        readonly />
+
+      <button class="button button--no-margin button--copy js-copy-url-button" data-clipboard-target="#share-input">Copy</button>
+    </div>
+
+    <p><a href="mailto:?subject=<%= URI.escape(@research_session.name) %>&body=<%= URI.escape(request.original_url) %>" target="_blank">You can email this link by clicking here</a></p>
```

Comment:
Could we keep the copy here to a call to action "Email this link"? "Click[ing] here" feels like a copy smell.

Reply:
(and if we do that, could we also remove the "You can" from the "bookmark" part?)